### PR TITLE
Update Publish Data section

### PR DESCRIPTION
**Pull request by fdelavega**
@jmcanterafonseca Update the publish data section to replace WStore data publication by Biz Ecosystem data publication

### CODE REVIEW

**Comment by jmcanterafonseca** [comment]
thanks @fdelavega  overall looks good but some improvements over the text are needed ...

**Comment by fdelavega** [comment]
@jmcanterafonseca I have made a general review of the whole section, let me know if you need any other change

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/acquiring-datasets-offered-in-the-wstore.md [hunk 1]
```diff
@@ -1,41 +1,64 @@
...
+FIWARE Account. Once in the FIWARE Store, you can search for CKAN datasets
+filtering by category. In particular, you can browse categories using the
+the breadcrumb located in the top of the offering list, and search for
+the “Datasets” category.
+
+[![Acquiring Datasets Offered in the
+FIWARE Store](../images/Acquiring-Datasets-Offered-in-the-WStore.png)](../images/Acquiring-Datasets-Offered-in-the-WStore.png)
+
+Once you have selected the Data category, all the Product Offerings where a dataset is
+offered are displayed. Therefore, you can choose those ones that fits
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/acquiring-datasets-offered-in-the-wstore.md]: nit: fit

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/acquiring-datasets-offered-in-the-wstore.md [hunk 2]
```diff
@@ -1,41 +1,64 @@
...
+your requirements and add them to the shopping cart by clicking on “Add to Cart”.
 
 [![Acquiring Datasets Offered in the
-WStore](../images/Acquiring-Datasets-Offered-in-the-WStore.png)](../images/Acquiring-Datasets-Offered-in-the-WStore.png)
+FIWARE Store1](../images/Acquiring-Datasets-Offered-in-the-WStore1.png)](../images/Acquiring-Datasets-Offered-in-the-WStore1.png)
+
+By clicking this button, a new dialog is displayed in order to
+allow you to select characteristics, if available, and the price
+plan. In addition, if the offering has some legal conditions attached,
+you will be forced to accept them before completing the process.
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/acquiring-datasets-offered-in-the-wstore.md]: you will have to

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/acquiring-datasets-offered-in-the-wstore.md [hunk 3]
```diff
@@ -1,41 +1,64 @@
...
 
 [![Acquiring Datasets Offered in the
-WStore4](../images/Acquiring-Datasets-Offered-in-the-WStore4.png)](../images/Acquiring-Datasets-Offered-in-the-WStore4.png)
+FIWARE Store6](../images/Acquiring-Datasets-Offered-in-the-WStore6.png)](../images/Acquiring-Datasets-Offered-in-the-WStore6.png)
 
-A new dialog will be displayed with all the resources contained in the
-offering. Among these resources, the user will find the dataset:
+In the details of the product, you can see several information regarding
+the acquisition, including the characteristics and the price model you have
+selected, and the list of charges you have made (Including its invoice).
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/acquiring-datasets-offered-in-the-wstore.md]: nit (including ...)

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/introduction.md [hunk 4]
```diff
@@ -16,21 +16,21 @@ the data in the FIWARE Store since you will also get the possibility of:
 -   **Charging the users for accessing your data**: there are some
     conditions (conflicts of interest, security, competence...) that
     force you to publish data with cost. In that cases, you can use CKAN
-    and the Store together to charge the users for using your data. You
-    are able to choose different payment models:
-    -   **Single payment:** users pay once.
-    -   **Subscription:** users are forced to pay periodically (daily,
-        monthly,...) for using your data. In addition, users will be
+    and the FIWARE Store together to charge the users for using your data.
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/introduction.md]: to charge users

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-directly-through-the-ckan-interface.md [hunk 5]
```diff
@@ -1,15 +1,44 @@
...
+Note that you can use the “License” and “License Description” fields in order
+to include the terms and conditions for using the data. In this regard,
+if you include those fields, customers will need to explicitly accept them
+to be able to acquire the offering.
 
 [![Offering Datasets Directly Through the CKAN
 Interface1](../images/Offering-Datasets-Directly-Through-the-CKAN-Interface1.png)](../images/Offering-Datasets-Directly-Through-the-CKAN-Interface1.png)
+
+You may have noticed that this way of creating offerings has some restrictions.
+On the one hand, the offering is created just with the dataset, so you cannot
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-directly-through-the-ckan-interface.md]: On one hand

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md [hunk 6]
```diff
@@ -1,97 +1,185 @@
...
-    open resources. (Typically you should disable this checkbox).
-
-​[![Offering Datasets Through the WStore
+As mentioned before, you can also offer your CKAN datasets in the FIWARE
+Store by using the methods provided by the Store itself. Using the FIWARE Store
+interface for offering your CKAN datasets allows you to create more advanced
+offerings, including composite ones, offerings with multiple pricing models,
+offerings with recurring and usage pricing models, etc. As an alternative,
+you can use the CKAN interface to create a basic offering, and then use
+the FIWARE Store interface to include these advance features.
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md]: nit: advanced

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md [hunk 7]
```diff
@@ -1,97 +1,185 @@
...
+To create offerings, the first step is entering the [FIWARE
+Store](https://store.lab.fiware.org/) and signing in using a valid FIWARE
+Account. Once in the FIWARE Store, click on the “My Stock” button in the left
+side bar in order to enter to the sellers section.
+
+[![Offering Datasets Through the FIWARE Store
+Interface](../images/Offering-Datasets-Through-the-WStore-Interface.png)](../images/Offering-Datasets-Through-the-WStore-Interface.png)
+
+Offering a dataset in the FIWARE Store requires the creation of two
+different entities. On the one hand, you have to create a Product Specification
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md]: on one hand

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md [hunk 8]
```diff
@@ -1,97 +1,185 @@
...
-[![Offering Datasets Through the WStore
+In this view, it is displayed a form distributed in several steps each
+intended to cover specific information of the Product Specification.
+Following it is described the purpose of these steps and the information
+you have to provide (Only the steps required for creating a CKAN Dataset
+are described, for more details have a look at the user guide in
+[here](http://business-api-ecosystem.readthedocs.io/en/latest/user-programmer-guide.html)):
+
+* **General**: This step is intended to provide the basic information of
+  the Product Spec. being created. In particular, you have to fill the
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md]: product specification

Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md]: you have to fill in

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md [hunk 9]
```diff
@@ -1,97 +1,185 @@
...
+  following fields:
+
+    -   Name: The name of your dataset (it doesn’t have to match with the name
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md]: does not

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md [hunk 10]
```diff
@@ -1,97 +1,185 @@
...
+    -   Brand: Your brand name (It migth be your username if you don’t have one)
+    -   ID Number: An external Id used for identify the dataset outside
+        of the FIWARE Store (i.e the Id in CKAN of the dataset)
+    -   Description: Description of the dataset
+
+[![Offering Datasets Through the FIWARE Store
+Interface2](../images/Offering-Datasets-Through-the-WStore-Interface2.png)](../images/Offering-Datasets-Through-the-WStore-Interface2.png)
+
+* **Assets**: This step is intended to provide information of the asset
+  to be included as a Product Spec. In this case, as the CKAN dataset is
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md]: product specification

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md [hunk 11]
```diff
@@ -1,97 +1,185 @@
...
 
-In the following step you will be asked for the description, the pricing
-model and the legal terms. Using this form, you are only able to choose
-two pricing models: single payment or free offerings. If you want to set
-a more complex pricing model, you should provide a custom USDL. Relating
-to the legal conditions, you must note that all users will be forced to
-accept these conditions before acquiring the offering. If you don’t want
-users to accept legal terms, you can leave these fields in blank.
+* **Attachments**: This step is intended to provide the picture to be shown
+  in the Store portal regarding the Product Spec. To provide the picture
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md]: product specification

Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md]: fill in

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md [hunk 12]
```diff
@@ -1,97 +1,185 @@
...
+        will be displayed to the FIWARE Store customers
+    -   Places: List of places where you offering will be available
+
+[![Offering Datasets Through the FIWARE Store
 Interface7](../images/Offering-Datasets-Through-the-WStore-Interface7.png)](../images/Offering-Datasets-Through-the-WStore-Interface7.png)
 
-Finally, you must click in the “Publish” button to complete the process.
+* **Product Spec**: This step is intended to specify the Product Spec to
+  be included in the Product Offering. In this step you have to select
+  the dataset Product Spec you created previously.
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-through-the-wstore-interface.md]: product specification

File: doc/publishing-open-data-in-fiware/introduction.md [hunk 13]
```diff
@@ -1,12 +1,29 @@
...
+of applications and the creation of an innovation ecosystem. In this regard,
+this section explains how users (both end users and developers) can
+expose their data publishing it in CKAN, the Open Data publication GE.
+
+CKAN is an open source solution for the publication, management and
+consumption of open data, usually, but not only, through static
+datasets. CKAN allows to catalogue, upload and manage open datasets and
+data sources, while supports searching, browsing, visualizing or
+accessing open data. CKAN is the Open Data publication platform that is
+most widely used by many cities, public authorities and organizations.
```
Comment (jmcanterafonseca) [inline doc/publishing-open-data-in-fiware/introduction.md]: by cities

**Comment by jmcanterafonseca** [comment]
@fdelavega please check latest comments and be aware of the need of s/Product Spec/Product Specification/

thanks!

File: doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-directly-through-the-ckan-interface.md (hunk 5 above)
Comment (fdelavega) [inline doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-directly-through-the-ckan-interface.md]: On the one hand .. On the other hand, is a correct structure
http://dictionary.cambridge.org/es/diccionario/ingles/on-the-one-hand-on-the-other-hand

"On one hand" is correct, but is an alternative form, not the preferred one
https://jakubmarian.com/on-the-one-hand-on-one-hand/

**Comment by fdelavega** [comment]
@jmcanterafonseca I have fixed the requested typos. I haven't modified two of your comments (see my comment in there)

**Comment by jmcanterafonseca** [comment]
LGTM, merged, thanks

**Pull request merged**